bootstrap/config/kfctl_aws.yaml

```diff
@@ -253,13 +253,14 @@ spec:
   packageManager: kustomize
   repos:
     - name: kubeflow
-      uri: https://github.com/kubeflow/kubeflow/archive/master.tar.gz
+      root: kubeflow-v0.6-branch
+      uri: https://github.com/kubeflow/kubeflow/archive/v0.6-branch.tar.gz
     - name: manifests
-      root: manifests-master
-      uri: https://github.com/kubeflow/manifests/archive/master.tar.gz
+      root: manifests-v0.6-branch
```

Comment:
I downloaded the file from the uri, the dir name is `manifests-0.6-branch`.
(No v)

Reply:
Thanks for catching this.
Seems really weird that github would change the pattern for this branch.
I thought the naming was supposed to always be: `<repo>-<branch>`